src/main/java/com/snowflake/kafka/connector/internal/streaming/StreamingClientHandler.java

```diff
@@ -50,34 +65,42 @@ public static boolean isClientValid(SnowflakeStreamingIngestClient client) {
     return client != null && !client.isClosed() && client.getName() != null;
   }
 
+  public static String getLoggableClientProperties(Properties properties) {
+    return properties.entrySet().stream()
+        .filter(
+            propKvp ->
+                LOGGABLE_STREAMING_CONFIG_PROPERTIES.contains(
+                    propKvp.getKey().toString().toUpperCase()))
+        .collect(Collectors.toList())
+        .toString();
```

Comment:
this string is essentially returning only the keys? is that what you want or you wanted key value pair both?

Reply:
No this string returns keys and values. The filter method only looks at the string, but the collect will contain the entire key value pair. Added a test in StreamingClientHandler to confirm this